LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 ANSYS, Inc. All rights reserved.
+Copyright (c) 2025 ANSYS, Inc. All rights reserved.
```

Comment:
@prmukherj This should be as follows

https://github.com/ansys-internal/pyansys-maintenance-automation/blob/main/scripts/file_references/LICENSE

![image](https://github.com/user-attachments/assets/b956ba27-8508-48b8-8ba5-bc7bd6198f85)

Reply:
2022 - 2025